headshots.py

```diff
@@ -1,19 +1,26 @@
-import cv2
+'''
+This code takes pictures of the user its recommended to take multiple per person
+'''
 import os
+import cv2
+import functions
 
 DATASET_FOLDER = "dataset"
-NAME = 'donkeykong' #replace with your name
+NAME = 'user2' #replace with your name
+CAMINDEX = 0
+
+cam = functions.camera_connection(CAMINDEX)
+
+if not os.path.exists(DATASET_FOLDER):
+    os.makedirs(DATASET_FOLDER)
 
-cam = cv2.VideoCapture(0)
+if not os.path.exists(DATASET_FOLDER+"/"+NAME):
+    os.makedirs(DATASET_FOLDER +"/"+ NAME)
 
 cv2.namedWindow("press space to take a photo", cv2.WINDOW_NORMAL)
 cv2.resizeWindow("press space to take a photo", 500, 300)
 
-if no os.path_exists(DATASET_FOLDER):
-    os.makedirs(DATASET_FOLDER)
-    
-if no os.path_exists(NAME):
-    os.makedirs(NAME)
+
 
 img_counter = 0
```

Comment:
pylint is insisting on treating this variable as a constant, so i recommend adding this comment to the end of the line: `# pylint: disable=invalid-name`